test/lib/ansible_test/_internal/commands/sanity/validate_modules.py

```diff
@@ -109,37 +134,48 @@ def test(self, args, targets, python):  # type: (SanityConfig, SanityTargets, Py
             else:
                 display.warning('Cannot perform module comparison against the base branch because the base branch was not detected.')
 
-        try:
-            stdout, stderr = run_command(args, cmd, env=env, capture=True)
-            status = 0
-        except SubprocessError as ex:
-            stdout = ex.stdout
-            stderr = ex.stderr
-            status = ex.status
+        errors = []
+        for plugin_type, plugin_targets in sorted(target_per_type.items()):
+            paths = [target.path for target in plugin_targets]
+            plugin_cmd = list(cmd)
+            if plugin_type != 'module':
+                plugin_cmd += ['--plugin-type', plugin_type]
```

Comment:
```suggestion

            if plugin_type != 'modules':
                plugin_cmd += ['--plugin-type', plugin_type]

```

Reply:
Fixed in 3673f6055ce08b11653dae65a76a35b84fe0f555.